setup/sql/5_3_hotfixes.php

```diff
@@ -106,4 +106,20 @@
 <#7>
 <?php
 $ilCtrlStructureReader->getStructure();
-?>
\ No newline at end of file
+?>
+<#8>
+<?php
+//removes org units from trash
+global $DIC;
+$obj_set = $DIC->database()->queryF('SELECT * FROM tree JOIN object_reference 
+ON tree.child=object_reference.ref_id JOIN object_data 
+ON object_reference.obj_id=object_data.obj_id 
+WHERE tree.tree < %s AND object_data.type = %s',array('integer', 'text'),array(0, "orgu"));
+$a_org_unit_ref_ids = array();
+while($row = $DIC->database()->fetchAssoc($obj_set))
+{
+	$a_org_unit_ref_ids[] = $row;
```

Comment:
`$row` is an array I think, therefore you should collect something like `$a_org_unit_ref_ids[] = $row['child']`. Please retest after.